ios/velo/AppDelegate.swift

```diff
@@ -29,40 +29,40 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // run so that the AstroViewController can spin up and get app.js executing.
         // This allows any splash screen presented to be displayed before the Launch Image
         // is dismissed (at the end of didFinishLaunchingWithOptions).
-        NSRunLoop.mainRunLoop().runUntilDate(NSDate().dateByAddingTimeInterval(1.0))
+        RunLoop.main.run(until: Date().addingTimeInterval(1.0))
```

Comment:
I think we can get rid of this whole line now that we do the `dismissLaunchImage`